symfony/phpunit-bridge/4.1/.env.test

```diff
@@ -0,0 +1,2 @@
+APP_ENV=test
+APP_SECRET=s$cretf0rt3st
```

Comment:
Is there any benefit to defining these? The `APP_ENV` will always already be set. Perhaps just leave it blank, but with a comment that overrides can go here?